data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            + " Giving Green attempts to solve a key question in the fight against the climate crisis but seems to be in a very early stage in doing so.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "Clean Air Taskforce",
+            link: "https://www.catf.us/",
+            keywords: ["ngo", "legislation", "energy", "technology"],
+            label: "catf",
+        },
+        description: {
+            content: "The CATF lobbies US governmental institutions to reduce emissions of greenhouse gases."
+            + " In addition they work to structure future [technical options for carbon-reduced energy production](https://www.catf.us/work/advanced-energy-systems/)."
+            + " CATF aims to assist corporates and governments in designing future energy systems based on renewables, biofuels, nuclear and systemic improvements of grids and consumption.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "ImpactMatters - Climate Change",
+            link: "https://www.impactmatters.org/top-lists/climate-change/",
+            keywords: ["ngo", "impact-assessment", "tree-planting"],
+            label: "impact-matters",
+        },
+        description: {
+            content: "Here, Impact Matters analyses the efficiency of tree planting initiatives."
+            + " It is an attempt to provide donors with the most efficient ways of fighting climate change."
+            + " While tree planting is important, changing public opinion and governments is arguably more impactful."
+            + " Also they seem to rely on impact statements by the organizations they list. There seems to be no independent scientific assessment." 
+            + " That makes initiatives overstating their impact more likely to receive funding.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+    },
+    {
+        meta: {
+            name: "Rainforest Foundation US",
+            link: "https://rainforestfoundation.org",
+            keywords: ["ngo", "rainforest", "conservation", "indigenous-communities"],
+            label: "rainforest-foundation",
+        },
+        description: {
+            content: "The Rainforest Foundation works on-the-ground to secure land rights for indigenous people."
+            + " They train indigenous communities to use technology to protect their forests."
+            + " The Foundation invests directly in indigenous communities and provides them with the tools, training, and resources necessary to protect their rainforests."
+            + " So far they protected more than 134 km² of rainforest.",
+            updatedBy: "Alexander", updatedAt: "2020-02-16"
+        },
+        actions: {
+            content: "Supply legal and technical support to allow indigenous communities to make decisions and influence public policy"
```

Comment:
- Supply

Reply:
Not sure I understand your comment

Reply:
sorry, I meant - Supply
Or is this sentence more likely a heading for the upcomming bullet points? Then a colon would be great for orientation